mkdocs/pages/en/devbook/accounts/create-from-mnemonic.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Create from Mnemonic
+---
+
+# Creating Accounts from Mnemonics
+
+This tutorial shows how to create <accounts:> for the Symbol blockchain using a <mnemonic phrase:>.
```

Comment:
```suggestion
This tutorial shows how to create <accounts:> for the Symbol blockchain using a <mnemonic phrase:>,
also known simply as _mnemonic_.
```

Because the rest of the tutorial just calls it mnemonic.